libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/SalesforceSDKManager.m

```diff
@@ -39,6 +39,7 @@
 #import "SFSDKNullURLCache.h"
 #import "UIColor+SFColors.h"
 #import "SFDirectoryManager+Internal.h"
+#import <SalesforceSDKCore/SalesforceSDKCore-Swift.h>
```

Comment:
Could we use `#import "SalesforceSDKCore-Swift.h"` since this is within the same library?